src/adapters/Logary.Adapters.Facade/Logary.Adapters.Facade.fs

```diff
@@ -195,6 +195,8 @@ module LogaryFacadeAdapter =
             field.PropertyType,
             fun _ -> Date.timestamp () |> box)
 
+        | "consoleSemaphore" ->
+          obj() // Todo: should this come from Logary now? where?
```

Comment:
From https://github.com/logary/logary/blob/master/src/Logary/Internals_Globals.fs#L8 as a new object. Where's Logary's official colourised target btw? I was gonna point to it but ... ;)